addons/website_event_exhibitor/static/src/js/event_sponsor_search.js

```diff
@@ -10,6 +10,7 @@ publicWidget.registry.websiteEventSearchSponsor = publicWidget.Widget.extend({
         'click .o_dropdown_reset_tags': '_onTagReset',
         'change .o_wevent_event_tags_form input': '_onTagAdd',
         'change .o_wevent_event_tags_mobile_form input': '_onTagAddMobile',
+        'search .o_wevent_event_search_box': '_onSearch',
```

Comment:
Hello @thsh-odoo :wave: :smile: 
I tested and found that this line is enough to fix the 2 issues (search on 'enter' and cancel the search by clicking on the cross). Could you take a look to see if I'm not mistaken? :smile: 

Also both issues are present since version 15.0 so could you retarget the PR? :slightly_smiling_face: 

Thanks and cheers!

Reply:
Hello @amdi-odoo 
I didn't get the first point can you please explain it in detail :sweat_smile: 
I haven't targeted it to 15.0 because this is the sub-task of the task mentioned in the log and on that task AWA told me to make a clean fix for the master if I am not mistaken :smiley:

Reply:
Well I searched but couldn't find an alternative so let's go for it :slightly_smiling_face: 
Thanks!